### ISSUE TITLE
Learner practice quiz report - redundant navigation error on icons for question attempts

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8813]
## Observed behavior
In the Learner's report for a practice quiz (`ReportsLessonLearnerExercisePage`), there's bug around navigation when a user clicks on the icons for the different attempts more than once (or, if a user has only one attempt that is incorrect, it seems to give this error on the first click - as shown in the gif below).
![2021-12-06 15 01 00](https://user-images.githubusercontent.com/13563002/144936712-20af243a-bc4a-4d8d-808a-3e31cf35beab.gif)

## Errors and logs
In browser console:
```
Uncaught (in promise) Error: Avoided redundant navigation to current location: "/d7d5b2bae1fa62db5590c891e1256e4c/reports/lessons/aacfecd6f9d041adab0c2e4ad3025bab/learners/e713102e3fe0a3da24f4c2d5de8e7805/exercises/37400c4cdcbb5abd941fecbc18207a62/try/0/questions/0/interactions/0".
    at createRouterError (vue-router.esm.js:2062)
    at createNavigationDuplicatedError (vue-router.esm.js:2035)
    at HashHistory.confirmTransition (vue-router.esm.js:2195)
    at HashHistory.transitionTo (vue-router.esm.js:2125)
    at HashHistory.push (vue-router.esm.js:2595)
    at vue-router.esm.js:2913
    at new Promise (<anonymous>)
    at VueRouter.push (vue-router.esm.js:2912)
    at VueComponent.handleNavigation (ReportsLessonLearnerExercisePage.vue:41)
    at invokeWithErrorHandling (vue.runtime.esm.js:1863)
```

## Expected behavior
No error in console.

## Steps to reproduce
1. Sign in as coach and go to reports 
2. Get a report for an individual learner on a practice quiz (currently, this is only working if there has been one completed attempt of the practice quiz - multiple practice quizzes are not currently loading reports)
3. Click on the icon in the answer history, and note the redundant navigation error.

## Context

- Kolibri v0.15.x
- Chrome

**Comment by rtibbles** [comment]
Hrm - perhaps we should be disabling already active interactions?

**Comment by sairina** [comment]
Just checked - can verify that this is also an issue for when a learner takes a regular quiz in 0.15 and looks at the report for a quiz.

**Comment by rtibbles** [comment]
Fixed in #8521
